docs/INSTALLATION.md

```diff
@@ -1,13 +1,15 @@
 ## Dart
-* [Install Dart](https://www.dartlang.org/install)
-* Add the Dart SDK to your path.
-* From a new command window, type `dart --version` to verify that Dart is now available.
+* One option is to follow the [instructions for installation on the Dart website](https://www.dartlang.org/install).
 
-## Dart Plugin for your IDE or Editor
-* If you are using an IDE or an editor, [install and configure the Dart plugin](https://www.dartlang.org/tools).
+### Manual install instruction
+* Download a zip file from [this page](https://www.dartlang.org/install/archive).
+* Unzip the file into whatever directory you choose.
+* Add the full path of that directory to your PATH.
+    * Follow the links below for updating your PATH on Windows, Linux and Mac platforms:
+        * [Instructions for Windows](https://www.computerhope.com/issues/ch000549.htm)
+        * [Instructions for Linux](https://www.computerhope.com/issues/ch001647.htm)
+        * [Instructions for Mac](https://www.pegaxchange.com/tag/macos-sierra-add-entry-to-path-variable/)
+* From a new command window or terminal, type `dart --version` to verify that Dart is now available. If the output shows the version number, then Dart is correctly installed on your computer.
```

Comment:
@Stargator Yes it does. And it's clear too. But English is not my native language and this is one of the many mistakes I used to make when I was starting out. Even though the instruction says to start a new terminal I used to continue using the same one. Also note that this instruction is in continuation with 3 links for 3 platforms, a student might just use the same terminal thinking its a continuous process.

So I suggest to add a section called "Verifying dart installation". This verifying of installation is helpful in both automatic and manual installations.   

The significant word in our instruction is '**new** ' it's very small and can be easily missed. So another suggestion is instead of adding a new section maybe just highlight the word to emphasize its importance. 

What do you think?